internal/ansi/spinner.go

```diff
@@ -10,10 +9,7 @@ import (
 
 const (
 	spinnerTextEllipsis = "..."
-	spinnerTextDone     = "done"
-	spinnerTextFailed   = "failed"
-
-	spinnerColor = "red"
+	spinnerColor        = "red"
```

Comment:
Can we make it a color more similar to the indigo (#635DFF) that is one of the 2 primary colors of the new brand?

Reply:
Sure. Just have to check it's supported by the lib.

Reply:
https://github.com/briandowns/spinner/blob/master/spinner.go#L35 these are the only colors

Reply:
Cyan, then. It's the closest.

Reply:
<img width="692" alt="Screen Shot 2021-03-19 at 21 45 21" src="https://user-images.githubusercontent.com/5055789/111854120-6d310f80-88fc-11eb-8d0a-966a141032b4.png">